src/AWS.Deploy.CLI/Utilities/CommandLineWrapper.cs

```diff
@@ -36,6 +36,7 @@ public class CommandLineWrapper : ICommandLineWrapper
             bool streamOutputToInteractiveService = true,
             Action<TryRunResult> onComplete = null,
             bool redirectIO = true,
+            string awsExecutionEnv = "",
```

Comment:
should `awsExecutionEnv` really be passed in as a method parameter?  Wouldn't it always be the same for the lifetime of the `CommandLineWrapper`?  Would it make more sense to treat it like `_awsCredentials` or `_awsRegion` and pass it to the constructor?

Reply:
Looked over usage, you're passing in Recipe details, which aren't known until after the user selects a recipe.  

Hmm ... still feels like an odd method parameter on `ICommandLineWrapper.Run` though.  But not sure what to advise.

Reply:
Storing a parameter is causing another problem of state management. I don't see saving this value to a shared object a workable solution.